packages/itwin/grouping-mapping-widget/src/widget/components/BlockingOverlay.tsx

```diff
@@ -13,8 +13,8 @@ export interface BlockingOverlayProps {
 
 export const BlockingOverlay = ({ isVisible }: BlockingOverlayProps) => {
   return (
-    <div className={classnames("group-mapping-blocking-overlay", isVisible && "visible")}>
-      <div className="group-mapping-blocking-overlay-spinner">
+    <div className={classnames("gmw-group-mapping-blocking-overlay", isVisible && "visible")}>
```

Comment:
I wonder if the `visible` class could conflict with anything. Maybe it should also be prefixed?

Reply:
Done for precaution